backend/inspirehep/records/api/literature.py

```diff
@@ -659,7 +657,7 @@ def merge_article_with_crossref_data(article):
 
     try:
         crossref_data = import_doi(doi)
-    except (ImportConnectionError, ImportParsingError):
+    except (requests.exceptions.RequestException, ImportParsingError):
```

Comment:
why we can't import it as `RequestException`?

Reply:
indeed just because we are already importing the whole module